_posts/ja/newsletters/2025-07-25-newsletter.md

```diff
@@ -0,0 +1,162 @@
+---
+今週のニュースレターでは、旧バージョンのLNDに影響する脆弱性の概要と、
+共同署名サービス利用時のプライバシーの向上策、量子耐性署名アルゴリズムへの移行が
+HDウォレットやスクリプトレスマルチシグ、サイレントペイメントに与える影響について掲載しています。
+また、Bitcoin Stack Exchangeで人気の質問とその回答や、新しいリリースとリリース候補の発表、
+人気のあるBitcoinインフラストラクチャソフトウェアの注目すべき更新に関する恒例のセクションも含まれています。
+
+## ニュース
+
+- **LNDのゴシップフィルターのDoS脆弱性:** Matt Morehouseは、
+  以前[責任を持って開示した][topic responsible disclosures]LNDの旧バージョンに影響する脆弱性について、
+  Delving Bitcoinに[投稿しました][morehouse gosvuln]。攻撃者は、
+  LNDノードがメモリ不足に陥って終了するまで、過去のゴシップメッセージを繰り返し要求することができました。
+  この脆弱性は、2024年9月にリリースされたLND 0.18.3で修正されました。
+
+- **<!--chain-code-withholding-for-multisig-scripts-->マルチシグスクリプトのためのチェーンコードの秘匿:**
+  Jurvis Tanは、Jesse Posnerと共同で行ったマルチシグ共同カストディのプライバシーとセキュリティ向上に関する研究について
+  Delving Bitcoinに[投稿しました][tan ccw]。一般的な共同カストディサービスでは、
+  以下の３つの鍵を用いた2-of-3のマルチシグが使用されます:
+
+  - ユーザーの _ホットキー_：ネットワーク接続されたデバイスに保存され、
+    ユーザーのためにトランザクションに署名します（手動またはソフトウェアにより自動で）
+
+  - プロバイダーのホットキー：プロバイダーが排他的に管理する別のネットワーク接続デバイスに保存されています。
+    この鍵は１日に _x_ BTCまでの支払いのみを許可するなど、ユーザが事前に定義したポリシーに従ってトランザクションに署名します。
+
+  - ユーザーの _コールドキー_：オフラインで保存され、ユーザーのホットキーが失われた場合、
+    またはプロバイダーが承認されたトランザクションへの署名を停止した場合にのみ使用されます。
+
+  上記の構成は、セキュリティを大幅に強化できますが、ほとんどの場合、
+  ユーザーがプロバイダーとユーザーのホットウォレットとコールドウォレットの
+  [BIP32拡張公開鍵][topic bip32]を共有するという設定方法が採用されています。
+  これにより、プロバイダーはユーザーのウォレットに入金されたすべての資金を検出し、
+  ユーザーがプロバイダーの支援なしに支払いをした場合でも、それらの資金のすべての支払いを追跡できます。
+  このプライバシーの損失を軽減する方法は、これまでにいくつか提案されていますが、
+  それらは通常の用途には適さない（例：個別のtapleafを使用する）か、
+  複雑（例：[MPC][]を必要とする）です。TanとPosnerは、シンプルな代替案を説明しています:
+
+  - プロバイダーはBIP32HD拡張鍵の半分（鍵部分）のみを生成し、公開鍵をユーザーに渡します。
+
+  - ユーザーは残りの半分（チェーンコード）を生成し、これは秘密にしておきます。
+
+  資金を受け取る際は、ユーザーは２つのパーツを組み合わせて拡張公開鍵（xpub）を作成し、
+  通常どおりマルチシグアドレスを導出します。プロバイダーはチェーンコードを知らないため、
+  xpubを導出したりアドレスを発見することはできません。
+
+  資金を使用する際は、ユーザーはチェーンコードから必要な _調整値_ を導出します。
+  プロバイダーは、この調整値を秘密鍵と組み合わせて有効な署名を作成します。
+  ユーザーは単にこの調整値をプロバイダーと共有するだけです。プロバイダーは、
+  調整値が資金を受け取った特定のscriptPubKeyからの支払いに有効であるということ以外、
+  調整値から何も知ることはできません。
+
+  一部のプロバイダーは、支払いトランザクションのお釣り用のアウトプットで、
+  資金を同じスクリプトテンプレートに送り返すよう要求する場合があります。
+  Tanの投稿では、これを簡単に実現する方法が説明されています。
+
+- **研究によりBitcoinの一般的なプリミティブが量子耐性署名アルゴリズムと互換性があることが示される:**
+  Jesse Posnerは、[量子耐性][topic quantum resistance]署名アルゴリズムが、
+  現在Bitcoinで使用されている[BIP32 HDウォレット][topic bip32]、[サイレントペイメントアドレス][topic silent payments]、
+  [スクリプトレスマルチシグアドレス][topic multisignature]、
+  [スクリプトレス閾値署名][topic threshold signature]と同等のプリミティブを提供することを示す研究論文のいくつかのリンクを
+  Delving Bitcoinに[投稿しました][posner qc]。
+
+## Bitcoin Stack Exchangeから選ばれたQ&A
+
+*[Bitcoin Stack Exchange][bitcoin.se]はOptech Contributor達が疑問に対して答えを探しに（もしくは他のユーザーの質問に答える時間がある場合に）アクセスする、
+数少ない情報ソースです。この月刊セクションでは、前回アップデート以降にされた、最も票を集めた質問・回答を紹介しています。*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Bitcoin Coreは、10ブロックを超える再編成をどのように処理しますか？]({{bse}}127512)
+  TheCharlatanは、最大10ブロック分のトランザクションのみをmempoolに再追加することでチェーンの再編成を処理する
+  Bitcoin Coreのコードのリンクを提供しています。
+
+- [暗号化ドライブと比較した署名デバイスの利点は何ですか？]({{bse}}127596)
```

Comment:
@azuchi seeing anchor related error in the log, didnt dig deep but perhaps this one?